package gunir.io;

import "gunir/io/column_metadata.proto";

message SchemaDescriptor {
    required string type = 1;
    required string description = 2;
    required string record_name = 3;
    optional string charset_encoding = 4; // input record's charset_encoding
}

message TabletSchema {
    required string table_name = 1;
    required SchemaDescriptor schema_descriptor = 2;
    repeated ColumnMetaData column_metadatas = 3;
}
